core/controllers/suggestion.py

```diff
@@ -481,6 +481,66 @@ class SuggestionListHandler(base.BaseHandler):
 
     GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
 
+    URL_PATH_ARGS_SCHEMAS = {}
+
+    HANDLER_ARGS_SCHEMAS = {
+        'GET': {
```

Comment:
For each of these fields, please link to a line of code in the frontend which shows that this handler can actually get called in practice with that argument.

(ALLOWED_SUGGESTION_QUERY_FIELDS might be a superset of what we actually use, and we should keep this list as tight as possible.)